test/socket-handle-error.js

```diff
@@ -1,72 +0,0 @@
-'use strict'
```

Comment:
Not sure how the hell I ended up removing this test. I'll restore it.

Reply:
It was removed in my PR that you rebased into here. I don't think it's relevant since we on longer use consume/unconsume.

Reply:
thnx